src/StackedTrait.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Atk4\Chart;
+
+/**
+ * Adds stacked chart functionality.
+ */
+trait StackedTrait
+{
+    /** @var array<mixed, array<string>> Stacks config - stack name => array of column names in stack */
+    protected $stacks = [];
```

Comment:
I was thinking that maybe it could be useful when you extend class with some additional functionality. But I guess you're right - no need for it (at least for now), removing.